nmdc_runtime/site/translation/submission_portal_translator.py

```diff
@@ -208,12 +198,25 @@ def _get_gold_study_identifiers(
         :param metadata_submission: submission portal entry
         :return: GOLD CURIE
         """
-        gold_study_id = get_in(["multiOmicsForm", "GOLDStudyId"], metadata_submission)
+        gold_study_id = get_in(["studyForm", "GOLDStudyId"], metadata_submission)
         if not gold_study_id:
             return None
 
         return [self._ensure_curie(gold_study_id, default_prefix="gold")]
 
+    def _get_ncbi_bioproject_identifiers(
+        self, metadata_submission: JSON_OBJECT
+    ) -> Union[List[str], None]:
+        """Construct a NCBI Bioproject CURIE from the multiomics from data"""
```

Comment:
Does the docstring in this internal method need to say "study form data" instead of "multiomics"?